site_scons/prereq_tools/base.py

```diff
@@ -675,11 +686,13 @@ def save_build_info(self):
     def __parse_build_deps(self):
         """Parse the build dependencies command line flag"""
         build_deps = GetOption('build_deps')
-        if build_deps in ('yes', 'only'):
-            self.download_deps = True
-            self.build_deps = True
-        elif build_deps == 'build-only':
+        skip_download = GetOption('skip_download')
+        if build_deps in ('fetch'):
```

Comment:
minor - this syntax actually doesn't do what it looks like
A single string in parens evaulates to a string. Whereas a single string in parens, with a comma, evaluates to a `tuple` with a single element
```
>>> ('fetch')
'fetch'
>>> ('fetch',)
('fetch',)
```
So notice both of these are `True`
```
>>> 'fetch' in ('fetch')
True
>>> 'fetch' in ('fetchplus')
True
```
but not this way
```
>>> 'fetch' in ('fetch',)
True
>>> 'fetch' in ('fetchplus',)
False
```